packages/x-charts/src/PieChart/index.tsx

```diff
@@ -1,5 +1,6 @@
 export { PiePlot } from './PiePlot';
 export { PieChart } from './PieChart';
-
-export * from './PieArc';
-export * from './PieArcLabel';
+export { PieArcLabel, getPieArcLabelUtilityClass, pieArcLabelClasses } from './PieArcLabel';
+export type { PieArcLabelClassKey, PieArcLabelClasses, PieArcLabelProps } from './PieArcLabel';
+export { PieArc, getPieArcUtilityClass, pieArcClasses } from './PieArc';
+export type { PieArcClassKey, PieArcClasses, PieArcProps } from './PieArc';
```

Comment:
Doing that to avoid breaking change, but not sure it's worth exporting them. For example, BarElements are not exported.

When doing cherry pick I would just remove the export